codegen/src/main/kotlin/software/amazon/smithy/rust/codegen/smithy/generators/EnumGenerator.kt

```diff
@@ -142,20 +131,20 @@ class EnumGenerator(
     }
 
     private fun renderEnum() {
+        val renamedWarning =
+            sortedMembers.mapNotNull { it.name() }.filter { it.renamedFrom != null }.joinToString("\n") {
+                val previousName = it.renamedFrom!!
+                "`$enumName::$previousName` has been renamed to `::${it.name}`."
+            }
         writer.docWithNote(
             shape.getTrait<DocumentationTrait>()?.value,
-            when (sortedMembers.any { it.collidesWithUnknown }) {
-                true ->
-                    "`$enumName::$UnknownVariant` has been renamed to `::$EscapedUnknownVariant`. " +
-                        "`$enumName::$UnknownVariant` refers to additional values that may have been added since " +
```

Comment:
Does the `Unknown` variant have documentation on it indicating what it is?

Reply:
yeah added (see description)